src/AllBookings/BookingCardsList.js

```diff
@@ -15,15 +15,16 @@ import { BookingState, type BookingStateType } from '../context/BookingState';
 type Props = {
   booking: BookingCardsList_booking,
   title: string,
+  dataCy?: string,
```

Comment:
what about wrapping this component by div in `src/AllBookings/index.js`, this component doesn't need to know about it.

Reply:
yep makes sense! tnx 😅